releasenotes/notes/nova-13.0-0c2497128e677cb2.yaml

```diff
@@ -0,0 +1,20 @@
+---
+prelude: >
+    This version introduces nova's new database,
+    nova api db.
+security:
```

Comment:
I do think this should be in security, but the note about speed should be here too instead of below in the other section